components/dialogs/AuthDialog.vue

```diff
@@ -725,7 +725,10 @@ export default {
       'loginByCosmosWallet',
     ]),
 
-    toggleFrontendMode,
+    toggleFrontendMode() {
+      logTrackerEvent(this, 'RegFlow', 'ToggleFrontendMode', 'ToggleFrontendModeLegacy)', 1);
```

Comment:
Do we know the exact mode the user is going to here? (`legacy` or `keplr/new`)

Reply:
Yes, the new UI is using `AuthDialogV2` component, I will add a tracker in it.

Reply:
so `ToggleFrontendModeLegacy` means from legacy to new?
then `ToggleFrontendModeXXX` will be added meaning from new to legacy?